src/index.js

```diff
@@ -429,6 +429,21 @@ f.analogWrite = function(pin, value, freq, callback) {
 
     // Make sure the pin has a PWM associated
     if(typeof pin.pwm == 'undefined') {
+          if(typeof pin.gpio != 'undefined' ){
+            if(value>=0.5)
+            {
```

Comment:
Let's try to align on style. We probably need to run everything through js beautifier. As of now, please keep curly braces on the same line as the opening statement. Also, please include a space before the opening brace.

Reply:
I have updated the patch by correcting the formatting issues , i did not notice these standards earlier , i will definitely make sure that i follow these formats in future.